bftengine/src/bftengine/RequestThreadPool.hpp

```diff
@@ -26,7 +26,7 @@ class RequestThreadPool final {
  public:
   static auto& getThreadPool() {
     static concord::util::ThreadPool threadBag{
-        ReplicaConfig::instance().get("concord.bft.bftengine.threadbagConcurrency", 16u)};
+        ReplicaConfig::instance().get("concord.bft.bftengine.threadbagConcurrency", 64u)};
```

Comment:
Thanks @yuliasherman for this comment. I will change this, but I want to know your view on the style of various config parameters. I can see we have 2 styles (we may have more, which I haven't yet deciphered):
1) Adding in ReplicaConfig as you mentioned.
2) The style for concord.bft.keyExchage.clientKeysEnabled 
Refer : https://github.com/vmware/concord-bft/blob/8432c3da710ef65ed4c12ef27cfb9b4aeec94e4e/bftengine/src/bftengine/KeyExchangeManager.cpp#L35
In style 2, the config is added directly in https://gitlab.eng.vmware.com/blockchain/vmwathena_blockchain/-/blob/master/concord/resources/bft_config.yaml
This is added if its needed, else its kept as it is if major changes are not required.

I think for frequently accessed config parameters adding as style (1) is good due to performance reasons, as we avoid the get in the config param map. 
But for one time used parameters (like threadbagConcurrency), style (2) is better as it reduces the effort of putting the config in yaml if not needed. Please share your valuable experience and feedback on this, so that I can course correct myself for any future configs.

Reply:
The second style is intended for parameters that could change their value during the run, which is not the case.

Reply:
Thanks @yuliasherman , I will go with style 1 in this case.